docs/spelling_wordlist

```diff
@@ -296,6 +296,7 @@ ip
 ipsum
 IPv
 IPython
+ipdb
```

Comment:
Could you sort this above "ipsum" on line 296, please.

Reply:
Done 👍